apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerDbType.cs

```diff
@@ -233,8 +233,7 @@ public override int GetHashCode()
         {
             unchecked
             {
-                int hashCode = StructMembers?.GetHashCode() ?? 0;
-                hashCode = (hashCode * 397) ^ (int) TypeCode;
+                int hashCode = (int) TypeCode;
```

Comment:
Oh I see. Missed that it was a dictionary. Perhaps maybe just including its Count in the hash code then so you get a bit better hash code distribution?